### ISSUE TITLE
Toolbar + immersive mode

### ISSUE THREAD

**Issue by nicolabeghin** [googlesamples/android-BasicImmersiveMode#1]
Would be useful to have examples referring to latest/suggested guidelines like having material Toolbar!

**Comment by nicolabeghin** [comment]
Any news about this? It's so frustrating to not have recent Android APIs examples. Developers are limited in adopting new android features if they need to spend their time hacking each new feature combination through stackoverflow threads

**Comment by nicolabeghin** [comment]
Any news on this?

**Comment by thakkaryash94** [comment]
Hey, I just port original immersive mode to material toolbar. Here is the [source link](https://github.com/thakkaryash94/ImmersiveModeSample). Hope this will help you.

**Comment by nicolabeghin** [comment]
Thanks @thakkaryash94 , I'll surely take a look! 
BTW would like to have some official samples from @googlesamples

**Comment by thakkaryash94** [comment]
True. I was also searching for it but then I decide to do it by myself as I wanted the same thing in my project.

**Comment by nicolabeghin** [comment]
up

**Comment by ingsaurabh** [comment]
@thakkaryash94 I think your sample dont work properly with supportActionBar, can you confirm
